trollflow2/tests/test_trollflow2.py

```diff
@@ -2015,5 +2049,88 @@ def fake_persist(*args):
     assert not sc_3a_3b["another"].attrs.get("persisted")
 
 
+yaml_test_s3_uploader_plain = """
+product_list:
+  output_dir: /tmp/
+  publish_topic: /topic
+  s3_config:
+    target: s3://bucket-name/
+  fname_pattern:
+    "{start_time:%Y%m%d_%H%M}_{platform_name}_{areaname}_{productname}.{format}"
+
+  areas:
+    euro4:
+      areaname: euro4
+      products:
+        airmass:
+          productname: airmass
+          formats:
+          - format: tif
+            writer: geotiff
+        natural_with_ir:
+          productname: natural_with_colorized_ir_clouds
+          formats:
+          - format: tif
+            writer: geotiff
+
+"""
```

Comment:
maybe put the s3 tests in another test module, eg `test_s3_upload.py`?

Reply:
Split to `test_s3_plugins.py` in https://github.com/pytroll/trollflow2/pull/158/commits/3505b3a071986eeef59295e4fee47861f1d1cb7f